osu.Game/Overlays/Mods/ModSelectScreen.cs

```diff
@@ -362,13 +372,26 @@ protected override void PopIn()
                 .FadeIn(fade_in_duration / 2, Easing.OutQuint)
                 .ScaleTo(1, fade_in_duration, Easing.OutElastic);
 
+            int nonFilteredColumnCount = 0;
+
             for (int i = 0; i < columnFlow.Count; i++)
             {
-                columnFlow[i].Column
-                             .TopLevelContent
-                             .Delay(i * 30)
-                             .MoveToY(0, fade_in_duration, Easing.OutQuint)
-                             .FadeIn(fade_in_duration, Easing.OutQuint);
+                var column = columnFlow[i].Column;
+
+                double delay = column.AllFiltered.Value ? 0 : nonFilteredColumnCount * 30;
+                double duration = column.AllFiltered.Value ? 0 : fade_in_duration;
```

Comment:
Is there a reason why this whole code is running if `column.AllFiltered == true`? That is, is there a reason why this couldn't be a precondition:
```
for (...)
{
    if (column.AllFiltered.Value)
        continue;

    ...

    nonFilteredColumnCount += 1;
}
```

Reply:
I've already touched on why this is being done this way [above](https://github.com/ppy/osu/pull/18157#issuecomment-1120437194).